src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/08-create-cluster-review.e2e-spec.ts

```diff
@@ -0,0 +1,61 @@
+import { CreateClusterWelcomePageHelper } from '../cluster/cluster-welcome-page.po';
+import { CreateClusterReviewPageHelper } from '../cluster/create-cluster-review.po';
+
+describe('Create Cluster Review page', () => {
+  const reviewPage = new CreateClusterReviewPageHelper();
+  const createCluster = new CreateClusterWelcomePageHelper();
+
+  beforeEach(() => {
+    cy.login();
+    Cypress.Cookies.preserveOnce('token');
+    createCluster.navigateTo();
+    createCluster.createCluster();
+
+    cy.get('.btn.btn-accent.m-2.float-right').click();
```

Comment:
IMHO we should start adding `id`'s to buttons (and other clickable elements).

Reply:
Definitely... Or using ARIA labels, which are exactly for describing the UI to visually impaired users.

Someone asked you: "Hey, how can I get to this address?" and you tell them: "Do you see that oil drop in the middle of the road? Ok, then count 3 bumps and 18 smashed insects and then turn right after the tire marks..." instead of simply reading the street names.

We have many ways to describe web elements, but style shouldn't be one of them (unless styles have any distinctive semantic meaning, which is not the case).

Reply:
> Definitely... Or using ARIA labels, which are exactly for describing the UI to visually impaired users.
> 
> Someone asked you: "Hey, how can I get to this address?" and you tell them: "Do you see that oil drop in the middle of the road? Ok, then count 3 bumps and 18 smashed insects and then turn right after the tire marks..." instead of simply reading the street names.
> 
> We have many ways to describe web elements, but style shouldn't be one of them (unless styles have any distinctive semantic meaning, which is not the case).

Done.